module/data/activity/damage-data.mjs

```diff
@@ -55,4 +55,22 @@ export default class DamageActivityData extends BaseActivityData {
     super.prepareFinalData(rollData);
     this.prepareDamageLabel(this.damage.parts, rollData);
   }
+
+  /* -------------------------------------------- */
+  /*  Helpers                                     */
+  /* -------------------------------------------- */
+
+  /** @override */
+  getDamageConfig(config={}) {
+    const rollConfig = super.getDamageConfig(config);
+
+    rollConfig.allowCritical ??= this.damage.critical.allow;
+    if ( this.damage.critical.bonus ) {
+      rollConfig.critical ??= {};
+      rollConfig.critical.bonusDamage = this.damage.critical.bonus;
+    }
+    rollConfig.critical ??= {};
```

Comment:
I suppose it would also be nice if we could use `rollConfig.critical.allow` instead of `rollConfig.allowCritical` but I guess `allowCritical` is what the damage roll config expects?

```suggestion
    rollConfig.critical ??= {};
    if ( this.damage.critical.bonus ) rollConfig.critical.bonusDamage = this.damage.critical.bonus;
```